demo/pages/2_👀_Manage_result.py

```diff
@@ -0,0 +1,56 @@
+import os
+import shutil
+
+import streamlit as st
+from utils.db_handling import delete_results, get_results
+from utils.state_handling import StateKeyEnum, check_state
+
+check_state(st.session_state)
+result_dir = st.session_state[StateKeyEnum.RESULT_DIR]
+
+st.cache_data.clear()
+results = get_results()
+if results is None or results.empty:
+    st.warning("No results found")
+
+
+else:
+    st.info("Review the results", icon="ℹ️")
+    st.dataframe(
+        results,
+        use_container_width=True,
+        hide_index=True,
+    )
+
+    test_id_1 = st.number_input(
```

Comment:
Is there a way to select the row in the dataframe?

Reply:
At the moment no, I'm hoping this feature gets implemented soon https://discuss.streamlit.io/t/selecting-a-single-row-while-using-st-data-editor-with-column-config-st-column-config-checkboxcolumn/48419/15